scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -230,25 +233,30 @@ const processRelatedAssetIds = async (
     })
   const manualRelatedAssetsResult = getManualRelatedAssetIds(assetId)
 
-  // ensure empty results get added so we can use this index to generate distinct asset list
```

Comment:
Maybe one too deep in the diff side of things 💀

Reply:
not bad at all